### PR TITLE
Spanish translation completed

### PR DESCRIPTION
**Pull request by norbux**
This PR includes all the necesary files traslated to Spanish

content/download/_index.es.md
content/learn/_index.es.md
content/learn/getting-started.es.md
content/learn/overview.es.md
content/learn/samples.es.md
content/learn/tools.es.md
content/learn/why_zig_rust_d_cpp.es.md
content/news/_index.es.md
content/zsf/index.es.md
content/_index.es.md
themes/ziglang-original/i18n/es.toml
config.toml

### CODE REVIEW

File: content/_index.es.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+</div>
+{{% /div %}}
+{{< /flexrow >}}
+<div style="height: 50px;"></div>
+
+{{< flexrow class="container" style="justify-content: center;" >}}
+{{% div class="main-development-message" %}}
+# Desarrollo principal
+El repositorio de Zig se encuentra en [https://github.com/ziglang/zig](https://github.com/ziglang/zig), donde también hospedamos el seguimiento a problemas y discuciones de propuestas.
+Contributors are expected to follow Zig's [Code of Conduct](https://github.com/ziglang/zig/blob/master/CODE_OF_CONDUCT.md).
```
Comment (kristoff-it) [inline content/_index.es.md:80]:
```suggestion
```

File: content/learn/_index.es.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Una lista de herramientas útiles al escribir código Zig.
+
+## Primeros pasos
+Si estas listo para comenzar a programar con Zig, esta guía te ayudará a poner en marcha el ambiente necesario.
+
+- [Primeros pasos]({{< ref "getting-started.md" >}})  
+
+## Recursos de aprendizaje en linea
+- [Aprende Zig](https://ziglearn.org)
+- Una introducción a Zig estructurada por [Sobeston](https://github.com/sobeston).
```
Comment (kristoff-it) [inline content/learn/_index.es.md:34]:
```suggestion
Una introducción a Zig estructurada por [Sobeston](https://github.com/sobeston).
```

This was an error in the english version that was recently fixed.

File: content/learn/overview.es.md [hunk 3]
```diff
@@ -0,0 +1,585 @@
...
+| tce | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A |
+| le | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A |
+| amdil | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A |
+| hsail | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A |
+| spir | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A |
+| kalimba | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A |
+| shave | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A |
+| renderscript | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A | [Tier 4](#tier-4-support) | [Tier 4](#tier-4-support) | N/A | N/A |
+
```
Comment (kristoff-it) [inline content/learn/overview.es.md:530]: The table above needs the chapter links to match what's linked by the table of contents. In your current version the links to the tier1/2/3/4 chapers are the following:

http://localhost:1313/es/learn/overview/#tier-1-support-soporte-nivel-1
http://localhost:1313/es/learn/overview/#tier-2-support-soporte-nivel-2
http://localhost:1313/es/learn/overview/#tier-3-support-soporte-nivel-3
http://localhost:1313/es/learn/overview/#tier-4-support-soporte-nivel-4

So the table must ling to the correct fragment (everything after the `#`).

E.g. `#tier-1-support` -> `#tier-1-support-soporte-nivel-1`

Comment (kristoff-it) [reply content/learn/overview.es.md:530]: Same is true also for the table below and other linked references to "Tier 1" etc. You can fix everything very quickly with a search and replace.

**Review by kristoff-it — CHANGES_REQUESTED** [review]
Thank you for your work!

I went ahead and left you some comments in the review, just a couple of minor things to fix.

I have two things that I have to point out that I could not put in the review:

1. You need to add Spanish in the translations page, so that users can navigate to it (`content/translations/index.md`)
2. Your branch still had a bunch of spurious commits because of the breaking changes that we did a while ago. I rebased this PR on top of latest master so now everything should be clean BUT you **need** to also fix your local copy on your machine. My recommedation, if you don't know how to do it and don't want to fuzz with git, is to just delete your local copy and clone your repo again from GitHub.

Let me know if you have issues with anything, I'm happy to help.

**Comment by kristoff-it** [comment]
No worries, I'm here to help you with that stuff :^)

**Comment by kristoff-it** [comment]
I asked @augur97 to take a look at the translation as I don't speak Spanish. It's meant to be a general review to spot typos and other basic mistakes more than anything. Don't stress too much over style and @norbux this is still your PR so the final call on any change is yours.

Thank you both for this work.

**Comment by norbux** [comment]
Excellent, welcome to the review @augur97

**Comment by norbux** [comment]
I posted a new pull request with the suggested fixes solved. I'll close this one so we can continue on the new PR for further conversation.

**Pull request closed without merging**